Themis/Views/Assessment/CorrectionSidebar/CorrectionGuidelinesCellView.swift

```diff
@@ -8,7 +8,14 @@
 import SwiftUI
 
 struct CorrectionGuidelinesCellView: View {
-    @ObservedObject var correctionGuidelinesModel: CorrectionGuidelines
+    @EnvironmentObject var assessment: AssessmentViewModel
+
+    var gradingCriteria: [GradingCriteria] {
+        if let criteria = assessment.submission?.participation.exercise.gradingCriteria {
+            return criteria
+        }
+        return []
```

Comment:
This feels like a good use case for `guard`:
```suggestion
        guard let criteria = assessment.submission?.participation.exercise.gradingCriteria else {
            return[]
        }
        return criteria
```
This way, it can be more easily be seen what's usually returned.